src/controllers/accounts/accounts.ts

```diff
@@ -248,6 +248,32 @@ export class AccountsController extends EventEmitter {
     await this.#storage.set('accounts', this.accounts)
   }
 
+  async reorderAccounts({ fromIndex, toIndex }: { fromIndex: number; toIndex: number }) {
+    if (
+      fromIndex < 0 ||
+      toIndex < 0 ||
+      fromIndex >= this.accounts.length ||
+      toIndex >= this.accounts.length
+    ) {
+      return this.emitError({
+        level: 'minor',
+        message: `Invalid fromIndex (${fromIndex}) or toIndex (${toIndex})`,
+        error: new Error('Invalid fromIndex or toIndex')
+      })
```

Comment:
Let's make this error major so that we can display an error toast to the user. Make the current message the message of the error and modify the message so it's human-readable - something along the lines of: "Failed to reorder accounts. Please reload the page and try again"